sevntu-checks/src/test/java/com/github/sevntu/checkstyle/checks/coding/Jsr305AnnotationsCheckTest.java

```diff
@@ -0,0 +1,714 @@
+        Assert.assertEquals(true, executed.get());
+        Assert.assertEquals(false, nullnessAnnotations.isEmpty());
+        Assert.assertEquals(true, nullnessAnnotations.contains(NullnessAnnotation.CHECK_FOR_NULL));
+        Assert.assertEquals(false, nullnessAnnotations.contains(NullnessAnnotation.NONNULL));
+        Assert.assertEquals(false, nullnessAnnotations.contains(NullnessAnnotation.NULLABLE));
+
+        nullnessAnnotations.clear();
+        executed.set(false);
+
+        final NullnessAnnotation[] annotationsToLookFor2 = new NullnessAnnotation[] {
+            NullnessAnnotation.CHECK_FOR_NULL,
+        };
+        check.new AbstractJsr305Check(ast) {
+            @Override
+            protected void runcheck() {
+                executed.set(true);
+                nullnessAnnotations.addAll(collectLookedForAnnotations(ast, annotationsToLookFor2));
+            }
+        }.check();
+        Assert.assertEquals(true, executed.get());
+        Assert.assertEquals(false, nullnessAnnotations.isEmpty());
+        Assert.assertEquals(true, nullnessAnnotations.contains(NullnessAnnotation.CHECK_FOR_NULL));
+        Assert.assertEquals(false, nullnessAnnotations.contains(NullnessAnnotation.NONNULL));
+        Assert.assertEquals(false, nullnessAnnotations.contains(NullnessAnnotation.NULLABLE));
+
+        nullnessAnnotations.clear();
+        executed.set(false);
+
+        final NullnessAnnotation[] annotationsToLookFor3 = new NullnessAnnotation[0];
+        check.new AbstractJsr305Check(ast) {
+            @Override
+            protected void runcheck() {
+                executed.set(true);
+                nullnessAnnotations.addAll(collectLookedForAnnotations(ast, annotationsToLookFor3));
+            }
+        }.check();
+        Assert.assertEquals(true, executed.get());
+        Assert.assertEquals(true, nullnessAnnotations.isEmpty());
+    }
+
+    private DetailAST createAstWithTwoAnnotations() {
+        final DetailAST ast = new DetailAST();
+        final DetailAST modifiersAst = new DetailAST();
+        modifiersAst.setType(TokenTypes.MODIFIERS);
+        ast.addChild(modifiersAst);
+        final DetailAST checkForNullAst = new DetailAST();
+        checkForNullAst.setType(TokenTypes.ANNOTATION);
+        final DetailAST checkForNullIdent = new DetailAST();
+        checkForNullIdent.setType(TokenTypes.IDENT);
+        checkForNullIdent.setText("CheckForNull");
+        checkForNullAst.addChild(checkForNullIdent);
+        modifiersAst.addChild(checkForNullAst);
+        final DetailAST nullableAst = new DetailAST();
+        nullableAst.setType(TokenTypes.ANNOTATION);
+        final DetailAST nullableIdent = new DetailAST();
+        nullableIdent.setType(TokenTypes.IDENT);
+        nullableIdent.setText("Nullable");
+        checkForNullAst.addChild(checkForNullIdent);
+        checkForNullAst.addNextSibling(nullableAst);
+        return ast;
+    }
+
+    @Override
+    protected String getPackageLocation() {
```

Comment:
Place override at the top of the class as the first method.

Reply:
Done.

Reply:
@mbert this isn't actually done.

Reply:
Maybe I misunderstood you. I moved the methods with `@Override` to the top of the class. Was that not what you asked me to do?